src/main/java/org/jmhsrobotics/frc2024/subsystems/arm/ArmClosedCommand.java

```diff
@@ -0,0 +1,30 @@
+package org.jmhsrobotics.frc2024.subsystems.arm;
+
+import org.jmhsrobotics.frc2024.controlBoard.ControlBoard;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class ArmClosedCommand extends Command {
+
+	ArmSubsystem armSubsystem;
+	ControlBoard control;
+
+	public ArmClosedCommand(ArmSubsystem armSubsystem, ControlBoard control) {
+		this.armSubsystem = armSubsystem;
+		this.control = control;
+		addRequirements(armSubsystem);
+	}
+
+	@Override
+	public void initialize() {
+		// TODO Auto-generated method stub
+		this.armSubsystem.setArmPivot(0);
+
+	}
+	@Override
+	public void execute() {
+		// TODO Auto-generated method stub
+		armSubsystem.setArmPivot(control.pitchInput());
+	}
+
+}
```

Comment:
This command has some problems. The setpoint is not properly scaled to degrees, also enabling the command forces the arm to twitch towards zero.